crates/abq_cli/tests/cli.rs

```diff
@@ -2727,6 +2728,299 @@ fn personal_access_token_does_not_mutate_remote_queue() {
     term(queue_proc);
 }
```

Comment:
This test above exercises the (PAT + explicit run ID) case, and was introduced when PAT support was added.